app/src/main/kotlin/com/wire/android/ui/settings/SettingsScreen.kt

```diff
@@ -1,37 +1,95 @@
-import androidx.compose.ui.text.font.FontWeight
-import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.res.stringResource
 import androidx.compose.ui.tooling.preview.Preview
-import androidx.compose.ui.unit.dp
-import androidx.compose.ui.unit.sp
+import androidx.hilt.navigation.compose.hiltViewModel
+import com.wire.android.R
+import com.wire.android.model.Clickable
+import com.wire.android.ui.common.ArrowRightIcon
+import com.wire.android.ui.common.RowItemTemplate
+import com.wire.android.ui.common.dimensions
+import com.wire.android.ui.debugscreen.ListWithHeader
+import com.wire.android.ui.theme.wireColorScheme
+import com.wire.android.ui.theme.wireTypography
 
-// the logic here is just temporary and will be updated to be added to the VM once we build the settings screen
 @Composable
-fun SettingsScreen() {
-    Column(
-        modifier = Modifier.fillMaxSize().wrapContentSize(Alignment.Center)
+fun SettingsScreen(settingsViewModel: SettingsViewModel = hiltViewModel()) {
+    val lazyListState = rememberLazyListState()
+
+    LazyColumn(
+        state = lazyListState,
+        modifier = Modifier.fillMaxWidth()
     ) {
-        Text(
-            text = "Settings is under construction",
-            fontWeight = FontWeight.Bold,
-            color = MaterialTheme.colorScheme.onBackground,
-            modifier = Modifier.align(Alignment.CenterHorizontally).padding(16.dp),
-            textAlign = TextAlign.Center,
-            fontSize = 18.sp
-        )
+        item {
+            ListWithHeader(stringResource(R.string.settings_general_label).uppercase()) {
+                settingsItem(stringResource(R.string.settings_your_account_label)) {}
+                settingsItem(stringResource(R.string.settings_app_settings_label)) {}
+                settingsItem(stringResource(R.string.settings_privacy_settings_label)) {}
+                settingsItem(stringResource(R.string.settings_network_settings_label)) {
+                    settingsViewModel.navigateToNetworkSettings()
+                }
+            }
+        }
+        item {
+            ListWithHeader(stringResource(R.string.settings_devices_label).uppercase()) {
+                settingsItem(stringResource(R.string.settings_manage_devices_label)) {}
+            }
+        }
+
+        item {
+            ListWithHeader(stringResource(R.string.settings_backups_label).uppercase()) {
+                settingsItem(stringResource(R.string.settings_back_up_label)) {}
+            }
+        }
+        item {
+            ListWithHeader(stringResource(R.string.settings_others_label).uppercase()) {
+                settingsItem(stringResource(R.string.settings_support_label)) {}
+                settingsItem(stringResource(R.string.settings_debug_settings_label)) {}
+                settingsItem(stringResource(R.string.settings_about_this_app_label)) {}
+            }
+        }
     }
 }
 
+@Composable
+fun settingsItem(text: String, onClick: () -> Unit) {
```

Comment:
i believe we have convention that Composable funs starts from the capital letter